content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -86,6 +86,12 @@ The `none` VM driver can result in security and data loss issues.
 Before using `--vm-driver=none`, consult [this documentation](https://minikube.sigs.k8s.io/docs/reference/drivers/none/) for more information.
 {{< /caution >}}
 
+Minikube also supports a `vm-driver=podman` similar to the Docker driver. Podman run as superuser privilege (root user) is the best way to ensure that your containers have full access to any feature available on your system.
```

Comment:
> Podman run as superuser privilege (root user) is the best way to ensure that your containers have full access to any feature available on your system.

@RA489, can you reword “best” to something like “a practical”? Or skip that sentence?